framework/core/Vcs.pm

```diff
@@ -231,7 +231,7 @@ sub contains_version_id {
 
   $vcs->checkout_vid(vid, work_dir)
 
-Performs a lookup of C<vid> in the C<commit-db> followed by a checkout of
+Performs a lookup of C<vid> in the C<active-bugs csv> followed by a checkout of
 the corresponding revision with C<revision_id> to F<work_dir>.
 Format of C<vid>: C<\d+[bf]>.
```

Comment:
Can you replace the "Format of vid ..." occurrences with a pointer to the Utils subroutine that checks for well-formedness?

Reply:
In the documentation? Should I replace `Format of C<vid>: C<\d+[bf]>` with `Format of C<vid> is checked using C<Utills::check_vid($vid)>`?

Reply:
Links added to Utils and Constants in the documentation.